skia-bindings/build_support/skia/config.rs

```diff
@@ -27,19 +28,34 @@ pub struct BuildConfiguration {
 
     /// C++ compiler to use
     cxx: String,
+
+    /// Optional target
+    pub target: Option<Target>,
 }
 
 /// Builds a Skia configuration from a Features set.
 impl BuildConfiguration {
     pub fn from_features(features: features::Features, skia_debug: bool) -> Self {
+        let cc = cargo::env_var("CLANGCC")
+            .or_else(|| cargo::env_var("CC"))
+            .unwrap_or_else(|| "clang".to_string());
+        let target = cc.find("--target=").and_then(|target_option_offset| {
+            cc[(target_option_offset + "--target=".len())..]
+                .split_once(' ')
```

Comment:
This assumes that a space follows `--target=XXX`, otherwise `None` is returned, and the target won't be extracted.

Reply:
Uhh, good point! It could be the last parameter. I'll fix that.